### ISSUE TITLE
Unhandled ORIN spec: zxY

### ISSUE THREAD

**Issue by danfossi** [time4tea/gopro-dashboard-overlay#52]
`wax@wks1 $ ~/venv/bin/gopro-dashboard.py ~/Desktop/GH010012.MP4 ~/Desktop/GH010012_Dash.MP4`

```
Starting gopro-dashboard version 0.50.0
Input file has size Dimension(x=1920, y=1080)
Timer(loading timeseries - Called: 1, Total: 0.82938, Avg: 0.82938, Rate: 1.21)
Timer(program - Called: 1, Total: 0.82941, Avg: 0.82941, Rate: 1.21)
Traceback (most recent call last):
  File "/home/wax/venv/bin/gopro-dashboard.py", line 98, in <module>
    gopro_frame_meta = framemeta_from(
  File "/home/wax/venv/lib/python3.10/site-packages/gopro_overlay/framemeta.py", line 239, in framemeta_from
    return parse_gopro(gpmd_from, units, metameta)
  File "/home/wax/venv/lib/python3.10/site-packages/gopro_overlay/framemeta.py", line 230, in parse_gopro
    accl_frame_meta = accl_framemeta(gopro_meta, units, metameta=metameta)
  File "/home/wax/venv/lib/python3.10/site-packages/gopro_overlay/framemeta.py", line 203, in accl_framemeta
    meta.accept(
  File "/home/wax/venv/lib/python3.10/site-packages/gopro_overlay/gpmd.py", line 311, in accept
    item.accept(visitor)
  File "/home/wax/venv/lib/python3.10/site-packages/gopro_overlay/gpmd.py", line 189, in accept
    i.accept(container_visitor)
  File "/home/wax/venv/lib/python3.10/site-packages/gopro_overlay/gpmd.py", line 189, in accept
    i.accept(container_visitor)
  File "/home/wax/venv/lib/python3.10/site-packages/gopro_overlay/gpmd.py", line 234, in accept
    getattr(visitor, method)(self)
  File "/home/wax/venv/lib/python3.10/site-packages/gopro_overlay/gpmd_visitors_xyz.py", line 52, in vi_ORIN
    self._orin = ORIN(item.interpret())
  File "/home/wax/venv/lib/python3.10/site-packages/gopro_overlay/gpmd_visitors_xyz.py", line 22, in __init__
    raise IOError(f"Unhandled ORIN spec: {conversion}")
OSError: Unhandled ORIN spec: zxY
```
For the tests I used the usual video (on the kart), the layout file is the standard one.

**Comment by time4tea** [comment]
Hiya. Thanks for raising this issue. Please could you let me know which type of gopro you have.

**Comment by danfossi** [comment]
This one is a GoPro Hero 8 Black

**Comment by time4tea** [comment]
It is easy enough to make it work for a new orientation. I can probably do that this evening.

**Comment by time4tea** [comment]
This is fixed in 0.51.0.
